apps/dev-api/bin/create-user.js

```diff
@@ -0,0 +1,50 @@
+const path = require('path')
+
+const { faker } = require('@faker-js/faker')
+
+const { prepareKeystoneExpressApp } = require('@open-condo/keystone/test.utils')
+
+const { User } = require('@dev-api/domains/user/utils/serverSchema')
+
+const BASIC_PHONE_REGEX = /^\+\d+$/
+
+function getJSON (opts) {
+    try {
+        return JSON.parse(opts)
+    } catch {
+        return undefined
+    }
+}
```

Comment:
IMO it is little excessively to define method for single usage. Here it's okay to put the content of this function inside `main`

Reply:
Just `Ctrl-C+V`'ed that from condo, but yeah...